examples/java/gradle-shared-module/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@examples/java-gradle-shared-module",
+  "version": "0.0.0",
+  "license": "MPL-2.0",
+  "scripts": {
+    "reinstall": "mvn install:install-file -Dfile=./../../../dist/java/com/hashicorp/cdktf/0.0.0/cdktf-0.0.0.jar -DgroupId=com.hashicorp -DartifactId=cdktf -Dversion=0.0.0 -Dpackaging=jar",
```

Comment:
```
    "reinstall": "mvn install:install-file -Dfile=./../../../dist/java/com/hashicorp/cdktf/0.0.0/cdktf-0.0.0.jar -DgroupId=com.hashicorp -DartifactId=cdktf -Dversion=0.0.0 -Dpackaging=jar",
```
Im not entirely sure what should be defined here or what part of the CI lifecycle `reinstall` is executed.

Reply:
reinstall, get, and synth are ran in sequence as part of the CI lifecycle.

Most examples rely on using a version that was just built. I'm thinking that this example may be more valuable showing what it's like to pull from GitHub, in which case this could be a no-op.

Reply:
I decided to leave it so that the example can also get tested against the local latest artifact version